ui/shared/components/panel/family/Family.jsx

```diff
@@ -31,81 +22,16 @@ import {
   FAMILY_FIELD_RENDER_LOOKUP,
   FAMILY_FIELD_OMIM_NUMBER,
   FAMILY_FIELD_PMIDS,
-} from '../../utils/constants'
-import { getAnalystOptions } from '../../../pages/Project/selectors'
-
-const FamilyGrid = styled(({ annotation, offset, ...props }) => <Grid {...props} />)`
```

Comment:
all the deleted code in this file was directly copy-pasted to `FamilyField.jsx` and `FamilyLayout.jsx`